src/codewhisperer/explorer/codewhispererNode.ts

```diff
@@ -74,13 +78,13 @@ export class CodeWhispererNode implements RootNode {
             return [createSsoSignIn(), createLearnMore()]
         }
         if (this._showFreeTierLimitReachedNode) {
-            if (isCloud9()) {
+            if (isCloud9() || isSageMaker()) {
```

Comment:
if this "pair" is found in many places, is there a common property that can encapsulate what's being checked, e.g.:
```suggestion
            if (hasVendedIamCredentials()) {
```

Reply:
If we do that, I would at least make `isSageMaker()` a separate callable just so that in case we have any SageMaker specific logics that don't apply to Cloud9.

Reply:
sounds good, will do